recipes/cairo/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1.2)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
Can this test package be converted to `test_package.c`?
This tests whether cairo can be used by c sources.